<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminClass</class>
 <widget class="QMainWindow" name="AdminClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="listWidgetAdmin"/>
    </item>
    <item>
     <widget class="QLineEdit" name="sizeLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="colourLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Colour</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="priceLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="quantityLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Quantity</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="photoLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Photograph</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateButton">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>506</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>AdminClass</receiver>
   <slot>addAdmin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>AdminClass</receiver>
   <slot>deleteAdmin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>AdminClass</receiver>
   <slot>updateAdmin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addAdmin()</slot>
  <slot>deleteAdmin()</slot>
  <slot>updateAdmin()</slot>
 </slots>
</ui>
